### ISSUE TITLE
Install not possible on hpc cluster

### ISSUE THREAD

**Issue by zutn** [philippkraft/cmf#19]
Installing on skylla throws error message:
```
pip install cmf --user --upgrade
Collecting cmf
  Downloading cmf-1.1.1.tar.gz (2.4MB)
    100% |████████████████████████████████| 2.4MB 225kB/s
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-build-6f6z0jeg/cmf/setup.py", line 211, in <module>
        updateversion()
      File "/tmp/pip-build-6f6z0jeg/cmf/setup.py", line 82, in updateversion
        doxycode = open('Doxyfile').readlines()
    FileNotFoundError: [Errno 2] No such file or directory: 'Doxyfile'

Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-build-6f6z0jeg/cmf/

```

**Comment by kbstn** [comment]
i can confirm that issue, 
on:
>>> cmf.__version__
'1.0.4a'

using:

pip install cmf --user --upgrade

**Comment by philippkraft** [comment]
Duplicate of #16, fixed for v1.2
